js/app.js

```diff
@@ -92,32 +54,79 @@ app.controller('CheckoutController', function($scope, $interval, Order, $httpPar
             };
             else
                 params = {};
-            params.finish_order = $scope.address;
+            params.finish_order = $scope.order_id;
             url = window.location.pathname;
             var serializedParams = $httpParamSerializer(params);
             if (serializedParams.length > 0) {
                 url += ((url.indexOf('?') === -1) ? '?' : '&') + serializedParams;
             }
             return url;
         }
-
-    //Create url for altcoin payment
-    $scope.alt_track_url = function(uuid) {
-        var params = getParameterByNameBlocko('wc-api');
-        if (params)
-            params = {
-                "wc-api": params
-            };
-            else
-                params = {};
-            params.uuid = uuid;
-            url = window.location.pathname;
-            var serializedParams = $httpParamSerializer(params);
-            if (serializedParams.length > 0) {
-                url += ((url.indexOf('?') === -1) ? '?' : '&') + serializedParams;
-            }
-            return url;
+    
+    // Select Blockonomics currency
+    $scope.select_blockonomics_currency = function(blockonomics_currency) {
+        $scope.currency = blockonomics_currency;
```

Comment:
The value is assigned to both `$scope.currency` and `blockonomics_currency;` and both are used in the code. I guess `$scope.currency`is used in frontend somewhere as its assigned to scope. Cleaner to use `$scope.currency` in all parts of code